nuget/framework/nunit.nuspec

```diff
@@ -30,6 +30,8 @@ Supported platforms:
       <group targetFramework="net45" />
       <group targetFramework="netstandard1.6">
         <dependency id="NETStandard.Library" version="1.6.0" />
+        <dependency id="System.Diagnostics.Process" version="4.3.0" />
+        <dependency id="System.Xml.XmlDocument" version="4.3.0" />
```

Comment:
Early comment: Both of these packages look like they're not _actually_ supported on iOS/Android. 😞 I haven't tried it, but unzipping the packages, they have those `_._` files...

Reply:
Conversation continued at https://github.com/nunit/nunit/pull/2586#issuecomment-348131519